src/openms/include/OpenMS/FILTERING/DATAREDUCTION/Deisotoper.h

```diff
@@ -47,6 +49,61 @@ class OPENMS_DLLAPI Deisotoper
 {
   public:
 
+  /** @brief Detect isotopic clusters in a mass spectrum.
+    
+    This algorithm is in parts taken from Guo Ci Teo et al, DOI: 10.1021/acs.jproteome.0c00544
+    and is closely related to deisotopeAndSingleCharge by Timo Sachsenberg.
+
+    Deisotoping is based on C13 abundance and will try to identify isotopic
+    clusters fitting to an averagine model, taking in account the corresponding
+    charge state. This only makes sense for peptide fragment ion spectra.
+    The algorithm considers each peak in the spectrum and will try to form
+    isotopic clusters for every charge state from @p min_charge to @p max_charge.
+    Of the clusters that pass an averaginge check based on KL-divergence (for all 
+    subclusters starting at the base peak as well), the largest one with the 
```

Comment:
at is `largest`? Number of peaks or total intensity explained?

Reply:
with respect to number of peaks. I changed the text 👍🏼